assets/airports/cyhz.json

```diff
@@ -0,0 +1,999 @@
+        "KIXIT": ["N44d48m33.00", "W063d08m31.00"],
+        "LEROS": ["N45d01m54.00", "W063d22m13.00"],
+        "LIRLA": ["N45d01m25.00", "W062d50m13.00"],
+        "LOGPO": ["N44d58m59.00", "W063d33m68.00"],
+        "MENVO": ["N44d53m86.00", "W063d11m68.00"],
+        "MIXIV": ["N45d02m10.00", "W063d35m80.00"],
+        "MODIR": ["N44d55m32.00", "W063d46m73.00"],
+        "MUVOX": ["N44d51m85.00", "W064d12m20.00"],
+        "NOTOP": ["N45d27m16.00", "W062d00m72.00"],
+        "NOTUB": ["N44d40m87.00", "W063d28m03.00"],
+        "NUBUK": ["N44d57m85.00", "W063d36m30.00"],
+        "NUTBY": ["N45d41m25.00", "W063d14m80.00"],
+        "OBNOM": ["N44d41m02.00", "W063d33m82.00"],
+        "ODKAS": ["N44d43m89.00", "W063d39m55.00"],
+        "OVTUV": ["N44d59m31.00", "W063d15m70.00"],
+        "PAULO": ["N45d15m20.00", "W063d20m10.00"],
+        "PENLU": ["N44d50m61.00", "W063d25m02.00"],
+        "PEPTA": ["N44d47m60.00", "W063d27m32.00"],
+        "PUTUD": ["N44d59m74.00", "W063d13m02.00"],
+        "PUTUX": ["N44d59m62.00", "W063d55m43.00"],
+        "RONPO": ["N44d40m34.00", "W063d51m58.00"],
+        "RUDBU": ["N44d51m87.00", "W063d23m11.00"],
+        "TAKIM": ["N44d52m47.00", "W063d28m74.00"],
+        "TETAR": ["N44d59m42.00", "W063d42m69.00"],
+        "TIDON": ["N44d59m78.00", "W063d31m13.00"],
+        "TUSKY": ["N43d33m90.00", "W067d00m00.00"],
+        "UDNIM": ["N44d47m75.00", "W063d07m17.00"],
+        "URMIN": ["N45d00m13.00", "W063d47m11.00"],
+        "VIMLA": ["N44d58m76.00", "W063d29m09.00"],
+        "VITAV": ["N44d57m49.00", "W063d38m81.00"],
+        "VOKIL": ["N44d47m15.00", "W063d18m13.00"],
+        "XOMRO": ["N44d42m93.00", "W063d40m38.00"],
+        "XOVMU": ["N44d55m97.00", "W063d35m76.00"],
+        "XUBKO": ["N45d03m84.00", "W063d19m77.00"],
+        "XUGID": ["N44d54m04.00", "W063d31m89.00"],
+        "YHZ"  : ["N44d55m40.00", "W063d24m10.00"],
+        "YQI"  : ["N43d49m51.00", "W066d04m95.00"],
+        "YQM"  : ["N46d11m34.00", "W064d34m25.00"],
+        "YZX"  : ["N44d55m63.00", "W065d05m12.00"]
+    },
+    "runways": [{
+        "name": ["05", "23"],
+        "end": [
+            ["N44d52m08.44", "W063d31m28.58", "449ft"],
+            ["N44d53m18.04", "W063d30m19.91", "443ft"]
+        ],
+        "ils": [false, true]
+    }, {
+        "name": ["14", "32"],
+        "end": [
+            ["N44d53m37.02", "W063d31m04.81", "435ft"],
+            ["N44d52m53.97", "W063d29m38.67", "476ft"]
+        ],
+        "ils": [true, false]
+    }],
+    "sids": {
+        "CYHZ4": {
+            "icao": "CYHZ4",
+            "name": "Halifax Four",
+            "rwy": {
+                "05": ["#053"],
+                "14": ["#143"],
+                "23": ["#233"],
+                "32": ["#323"]
+            },
+            "body": [],
+            "exitPoints": {
+                "CAAPE": ["ARDEE", "CAAPE"],
+                "VQI": ["HIDIG", "YQI"],
+                "NOTOP": ["IGTAS", "NOTOP"],
```

Comment:
I am still curious to see how we've determined the fixes that are worthy of being on the list of exit points or not. How did you get this information? How do you know it is the _complete_ truth and not just part of it? Also, unless there's a chart to back it up, the exit points should only be a single point, not a chain. Any subsequent fixes that are commonly used can be used in a route string for a given spawn pattern, but should not be listed as an exit point in the SID itself.

Reply:
So how am I actually meant to put these points in? There are _no_ official sources unless you contact navcanada who will most likely _not_ give that information